packages/admin/admin/src/dataGrid/excelExport/applyDefaultStyling.ts

```diff
@@ -0,0 +1,55 @@
+import { Worksheet } from "exceljs";
+
+export function applyDefaultStyling(worksheet: Worksheet) {
+    worksheet.eachRow((row, rowNumber) => {
+        row.height = 30;
+        for (let i = 1; i <= row.cellCount; ++i) {
+            const cell = row.getCell(i);
+
+            // Zebra
+            if (rowNumber % 2 === 1) {
+                cell.fill = {
+                    type: "pattern",
+                    pattern: "solid",
+                    fgColor: { argb: "FFf5f5f5" },
+                };
+            } else {
+                cell.fill = {
+                    type: "pattern",
+                    pattern: "solid",
+                    fgColor: { argb: "ffffffff" },
+                };
+            }
```

Comment:
This can be simplified and i don't think the comment adds any benefit to the readability of the code. 

```suggestion
            cell.fill = {
                type: "pattern",
                pattern: "solid",
                fgColor: { argb: rowNumber % 2 === 1 ? "FFf5f5f5" :"ffffffff" },
            };
```